### DISCUSSION TITLE
does watchtower take an existing Dockerfile or docker-compose.yml into account ?

### DISCUSSION THREAD

**Discussion by e-ferrari** [containrrr/watchtower#1556] [Q&A] [answered]
Hi,
i'm new to containers, so sorry if i ask something trivial. I hesitated for a long time to address containers because i always thought that updating containers is a difficult task. Now i found watchtower.
But what is if i  pull an image and use a Dockerfile or a docker-compose.yml to adapt the image to my needs ?
Does watchtower take an existing Dockerfile or docker-compose.yml into account ?
I read that it used the same options as the old image to start the updated image.
But what is with a Dockerfile or a docker-compose.yml ?

Thanks.

Bernd

**Comment by simonaronsson** [comment, marked as answer]
No, unfortunately not. The new container will inherit the configuration of the prior container.
